plugins/jobs/server/models/job.py

```diff
@@ -89,6 +89,19 @@ def list(self, user, types=None, statuses=None,
                                                 limit=limit, offset=offset):
             yield r
 
+    def listAll(self, limit=0, offset=0, sort=None, currentUser=None):
+        """
+        List all jobs.
+
+        :param limit: The page limit.
+        :param offset: The page offset
+        :param sort: The sort field.
+        :param currentUser: User for access filtering.
+        :deprecated: use list method instead
```

Comment:
I think the one we want for sphinx is `.. deprecated ::`

http://www.sphinx-doc.org/en/stable/markup/para.html#directive-deprecated